lambda/tmc.ml

```diff
@@ -644,8 +644,8 @@ let rec choice ctx t =
            not in tail-call position (after it returns
            we need to remove the exception handler),
            so it is not transformed here *)
```

Comment:
Nit: the "so it is not transformed here" part of that comment was the bug, and it should now be removed.

Reply:
Done, thanks.